Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -320,6 +320,23 @@ static NSString *const STPSDKVersion = @"13.0.3";
 
 @end
 
+#pragma mark Payment Intents
+
+/**
+ Stripe extensions for working with PaymentIntent objects.
+ */
+@interface STPAPIClient (PaymentIntents)
+
+/**
+ Retrieves the PaymentIntent object using the given secret. @see https://stripe.com/docs/api#retrieve_payment_intent
+
+ @param secret      The client secret of the payment intent to be retrieved. Cannot be nil.
+ @param completion  The callback to run with the returned PaymentIntent object, or an error.
+ */
+- (void)retrievePaymentIntentWithClientSecret:(NSString *)secret
```

Comment:
I'm wondering if there's less ambiguous vocabulary we can use so it's not confused with Stripe client secret? Or is it?

Reply:
looks like the rest of our documentation uses this ambiguous vocabulary so nevermind :)

Reply:
if we're confident it's not going to change, maybe we can mention it starts with `pi_*`